### PR TITLE
Version gitolite.rc in the gitolite-admin repo

### LINKED ISSUE #5: Version gitolite.rc and hooks
We should put `gitolite.rc` and repository hooks in the `gitolite-admin` repo, since:
1. The [privilege escalation](http://gitolite.com/gitolite/rc.html#privesc) issue doesn't exist: code execution as `git` only gives you access to the gitolite container.
2. This gives a sane way to configure gitolite, which we cannot currently do (except for poking directly at the data container)
3. This means gitolite's config is versioned.

This would enable doing #3 and #4 :-)

### PR DESCRIPTION
**Pull request by KellerFuchs**
Closes #5.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -25,6 +25,20 @@ if [ ! -e /srv/git/.gitolite ]; then
     echo "${GITOLITE_ADMIN}" > /tmp/admin.pub
     gitolite setup -pk /tmp/admin.pub
     rm /tmp/admin.pub
+
+    su -c 'git clone /srv/git/repositories/gitolite-admin.git /tmp/ga' git
+    su -c 'cp /srv/git/.gitolite.rc /tmp/ga/gitolite.rc'               git
+    su -c 'git -C /tmp/ga add gitolite.rc'			       git
+    su -c 'git -C /tmp/ga commit -m "Moving gitolite.rc to vcs"'       git
```
Comment (daurnimator) [inline run.sh:32]: What user is this commit created as?

Comment (KellerFuchs) [reply run.sh:32]: `git`, and that should use the default thing (i.e. `<git@hostname>`)

**Comment by KellerFuchs** [comment]
Can someone test this independently of me?

**Comment by KellerFuchs** [comment]
@hashbang/administrators Ping?

**Comment by singlerider** [comment]
@kellerfuchs I'd love to test this for you. Can you tell me what you'd like to happen and how to execute a test?

File: run.sh [hunk 2]
```diff
@@ -25,6 +25,20 @@ if [ ! -e /srv/git/.gitolite ]; then
     echo "${GITOLITE_ADMIN}" > /tmp/admin.pub
     gitolite setup -pk /tmp/admin.pub
     rm /tmp/admin.pub
+
+    su -c 'git clone /srv/git/repositories/gitolite-admin.git /tmp/ga' git
+    su -c 'cp /srv/git/.gitolite.rc /tmp/ga/gitolite.rc'               git
+    su -c 'git -C /tmp/ga add gitolite.rc'			       git
```
Comment (daurnimator) [inline run.sh:31]: missing space for alignment?

**Comment by KellerFuchs** [comment]
`git-infra.#!` is going away in the near-ish future (it won't be necessary once we do the Ansible transition), so closing this.

**Pull request closed without merging**